pkg/bidstrategy/semantic/image_platform.go

```diff
@@ -1,25 +1,28 @@
-package docker
+package semantic
```

Comment:
I don't think this file should live in a general part of the code / I think that this being a Docker-specific strategy, it should live within the Docker executor package somewhere. My logic being that if/when we move to the world of executors being provided through plugins, then this strategy would need to be provided by the plugin and not by the core.

Reply:
Yeah that's fair. Although if there is ever more than one docker executor (i.e. and alternative implementation to our own) it would also need to implement something like this. My thinking here was that we could provide a library/package of "standard" bid strategies that other implementations could implement, mainly motivated by our mono repo approach today.

In ee288ce09c525fa8a5b7ea9bb53b60b9eff89856 I have moved this back into the docker executor.